strongbox-web-forms/src/main/java/org/carlspring/strongbox/forms/configuration/MavenRepositoryConfigurationForm.java

```diff
@@ -16,6 +20,11 @@
 
     private String cronExpression;
 
+    @Pattern(regexp = "checksum|refresh",
+            flags = Pattern.Flag.CASE_INSENSITIVE,
+            message = "metadataExpirationStrategy must contain one the following strings as value:  checksum, refresh")
```

Comment:
Although this will work just fine, I feel it can be improved by creating a generic `ValueValidator` that would get all of the values from the `Enum` class and would mark the field as invalid when the value is not found in the `Enum`. Thoughts @fuss86 ?

Reply:
I can do this.  Previously I had used a `@DescribableEnumValue`, but when I relocated `MavenStrategyEnum` into the Maven layout provider, it became no longer visible.  I found this approach used in some of the other forms, and went with it.  Let me know if I should create another enum visible to the forms classes for this.

Reply:
Yes, we should gain from `org.carlspring.strongbox.validation.configuration.DescribableEnumValue`.

@carlspring do you think we can depend on `strongbox-storage-maven-layout-provider` in `strongbox-web-forms` ? I think `strongbox-web-forms` may need some provider-specific validation sooner or later. Or we will need another provider-specific  `strongbox-web-forms` modules (like  `strongbox-maven-layout-provider-web-forms` )

Reply:
> @carlspring do you think we can depend on `strongbox-storage-maven-layout-provider` in `strongbox-web-forms` ? I think `strongbox-web-forms` may need some provider-specific validation sooner or later. Or we will need another provider-specific `strongbox-web-forms` modules (like `strongbox-maven-layout-provider-web-forms` )

I don't see a problem in this. I doubt it will cause some sort of cyclic dependency. Give it a go and let us know how it worked out. Thanks! :)